.github/tools/get_go_integration_tests.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
+import json
+
+
+if __name__ == "__main__":
+    import sys
+
+    tests_path = sys.argv[1]
+
+    test_files = [str(f).split("/")[-2] for f in Path(tests_path).glob("*/*_test.go")]
+    test_files = list(dict.fromkeys(test_files))
+    print(json.dumps(test_files))
```

Comment:
Can't we run `go list | grep /integrationtest/` instead of another python script?